iOS Example/iOS Example.xcodeproj/project.pbxproj

```diff
@@ -7,30 +7,34 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		7810985B1EAE77E900FC4CF2 /* RxSwift.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B388AAE91C65640700726BCA /* RxSwift.framework */; };
+		7810985C1EAE77E900FC4CF2 /* RxSwift.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = B388AAE91C65640700726BCA /* RxSwift.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
With Carthage, yes, you have to embed the framework in the app target & link from both 👍